engine/engine.go

```diff
@@ -421,7 +421,15 @@ func (r *ruleEngine) createViolation(conditionResponse ConditionResponse, rule R
 			if err != nil || codeSnip == "" {
 				r.logger.V(6).Error(err, "unable to get code location")
 			} else {
-				incident.CodeSnip = codeSnip
+				// Split the code snippet into lines
```

Comment:
I think all you need to do here is `incident.CodeSnip = fmt.Sprintf("%s\n", codeSnip)` If that doesn't fix it, probably newlines are not going to fix the issue